### PR TITLE
Improve parameter handling take 2

### PR DESCRIPTION
**Pull request by KaptenJansson**

### CODE REVIEW

**Comment by KaptenJansson** [comment]
Ping

**Comment by KaptenJansson** [comment]
cc @samdutton maybe you could take a look?

**Comment by samdutton** [comment]
LGTM from a quick look (about to get on a plane).

File: src/app_engine/apprtc.py [hunk 1]
```diff
@@ -560,16 +563,24 @@ def get(self, room_id):
         self.write_response('full_template.html')
         return
     # Parse out room parameters from request.
-    params = get_room_parameters(self.request, room_id, None, None)
+    loadingParams = get_room_parameters(self.request, room_id, None, None)
+    def maybe_include_loopback():
+      if loadingParams['isLoopback'] is True:
+        print 'here2'
+        return '<script src="/js/loopback.js"></script>'
```
Comment (andresusanopinto) [inline src/app_engine/apprtc.py]: This is ugly. Have the template do this if condition:

{% if include_loopback_js }
 <script src="/...."></script>
{% endif }

File: src/app_engine/apprtc.py [hunk 2]
```diff
@@ -219,21 +219,20 @@ def get_room_parameters(request, room_id, client_id, is_initiator):
...
-    warning_messages.append(message.encode('utf-8'))
+    warning_messages.append(message)
   if hd == 'true':
     video = 'mandatory:minWidth=1280,mandatory:minHeight=720'
   elif not hd and not video and get_hd_default(user_agent) == 'true':
     video = 'optional:minWidth=1280,optional:minHeight=720'
 
   if request.get('minre') or request.get('maxre'):
-    message = ('The "minre" and "maxre" parameters are no longer ' +
+    message = ('The &#34;minre&#34; and &#34;maxre&#34; parameters are no longer ' +
```
Comment (andresusanopinto) [inline src/app_engine/apprtc.py]: It is weird that you have to do this escape for warning_messages and  indicates there is probably a bug somewhere if this is needed.

**Comment by KaptenJansson** [comment]
Closing

**Pull request closed without merging**